workshops/templates/workshops/all_events.html

```diff
@@ -45,9 +45,9 @@
             </a>
           {% endif %}
         </td>
-        <td {% if not event.get_repository_url %}class="warning"{% endif %}>
-          {% if event.get_repository_url %}
-          <a href="{{ event.get_repository_url }}">{{ event.get_repository_url|cut:"https://github.com/" }}
+        <td {% if not event.repository_url %}class="warning"{% endif %}>
+          {% if event.repository_url %}
+          <a href="{{ event.repository_url }}">{{ event.repository_url|cut:"https://github.com/" }}
```

Comment:
Urk - no.  If we're going to massage URLs for display, we should do that in the view class, not in the template.  (Have a helper function to add another element to the context which is the value displayed?)  Reasoning is 1. logic all in one place and 2. we may allow a wider range of URLs in the very near future.